website/docs/docs/collaborate/environments/dbt-core-environments.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "dbt Core Environments"
+id: "dbt-core-environments"
+---
+
+## How do I maintain different environments with dbt Core?
```

Comment:
Hey @dave-connors-3, can you either pull this into the page title or remove it entirely? The reason is that when people google "dbt core environments" their search will return the title and the paragraph, but we'd rather only provide an informative paragraph.